### ISSUE TITLE
Make milestones and projects assignable through issue templates

### ISSUE THREAD

**Issue by ichwillkeinenaccount** [go-gitea/gitea#14629]
- Gitea version (or commit ref): 1.14.0+dev-665-g19fccdc45
- Git version:
- Operating system: docker image from docker hub
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
- Log gist:

## Description
I recently discovered the template feature for issues and created a test template with the same content as the documentation page. (https://docs.gitea.io/en-us/issue-pull-request-templates/) That worked.

Then I tried to assign the milestone and project (to get new issues to show up on the kanban board) through the template: 

```
---

name: "Template Name"
about: "This template is for testing!"
title: "[TEST] "
labels:

- bug
- "help needed"

milestones:
- 1.0.0
projects:
- test
---

This is the template!
```
I created the milestone and the project beforehand and tried it with "milestones"/"projects" as well as "milestone"/"project". Neither worked. 
I didn't find anything milestone or project related on the documentation page, in the github issues or via general google.

Is this even possible? If not, wouldn't it be a nice feature?

**Comment by lonix1** [comment]
I'm not sure about milestones.

But for "projects": being able to automatically add a new issue (user story) to the "backlog" on a kanban board would be nice. (What is currently referred to as "projects".)

**Comment by geospatialology** [comment]
I personally agree with milestones (and projects). I'm not bothered by recommitting a template or creating a new one. If milestones could be specified in a template, it means you can add a bunch (or even bulk load) tickets (tasks) that are associated with a planned release. Then once everything is complete, you can create a release associated with hitting the milestone. Often times when planning out software "next features", you know some (most/all occasionally) of the requirements and user stories and being able to queue these on a planned milestone more easily would be nice (i.e., don't have to select the milestone manually after creating the ticket - milestone already pre-populated by the template).
